src/components/Home/ChatBotIcon.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
+import NamePopUp from './NamePopUp';
+import BotIcon from '../../assets/amigoIcon.png';
+import ChatBox from '../ChatBox/index';
+const ChatBotIcon = () => {
+  const [namePopUp, setNamePopUp] = useState(false);
+  const [name, setName] = useState('');
+  const [displayChat, setDisplayChat] = useState(false);
+
+  return (
+    <div className='logo' style={{ right: 5, position: 'fixed', bottom: 5 }}>
```

Comment:
As you already using class here see if you can add this inline styles to external css.